api/docker/varnish/vcl/default.vcl

```diff
@@ -0,0 +1,89 @@
+vcl 4.0;
+
+import std;
+import xkey;
+import cookie;
+import var;
+
+include "./_config.vcl";
+include "./fos_tags_xkey.vcl";
+include "./fos_debug.vcl";
+
+sub vcl_recv {
+
+  var.set("originalUrl", req.http.X-Forwarded-Prefix + req.url);
+
+  if(var.get("originalUrl") ~ "^/api/varnish/healthcheck") {
+    return(synth(200,"OK"));
+  }
+
+  # Support xkey purge requests
+  # see https://raw.githubusercontent.com/varnish/varnish-modules/master/src/vmod_xkey.vcc
+  call fos_tags_xkey_recv;
+  
+  # exclude other services (frontend, print, etc.)
+  if (var.get("originalUrl") !~ "^/api") {
+    return(pass);
+  }
+
+  # exclude API documentation, profiler and graphql endpoint
+  if (var.get("originalUrl") ~ "^/api/docs" 
+    || var.get("originalUrl") ~ "^/api/graphql" 
+    || var.get("originalUrl") ~ "^/api/bundles" 
+    || var.get("originalUrl") ~ "^/api/contexts" 
+    || var.get("originalUrl") ~ "^/api/_profiler" 
+    || var.get("originalUrl") ~ "^/api/_wdt") {
+    return(pass);
+  }
+
+  # exclude any format other than HAL
+  if (req.url !~ "\.jsonhal$" && req.http.Accept !~ "application/hal\+json"){
+    return(pass);
+  }
+}
+
+sub vcl_hash {
+  if (req.http.Cookie) {
+    # Include JWT cookies in cache hash 
+    cookie.parse(req.http.Cookie);
+    cookie.keep("localhost_jwt_hp,localhost_jwt_s");
```

Comment:
why is this line here?

Reply:
Good catch. This should use COOKIE_PREFIX instead of localhost_

Reply:
Fixed with https://github.com/ecamp/ecamp3/pull/3610/commits/60ce02fcd9a335e8c0e9ce4dc8bfe54bf7932bca